.github/workflows/pr_test.yml

```
@@ -0,0 +1,76 @@
+on: pull_request
+name: Test
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v1
+    - name: Install Go
+      uses: actions/setup-go@v1
```

V2 has now been released and fixes the GOPATH issue we had a workaround for in the CLI, which means we can also remove the shell step below. 

```suggestion
      uses: actions/setup-go@v2
```